cmd/image-builder/version.go

```diff
@@ -46,6 +53,14 @@ func readVersionInfo() *versionDescription {
 		}
 	}
 
+	cmd := exec.Command(osbuildCmd, "--version")
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	if err := cmd.Run(); err != nil {
+		vd.ImageBuilder.Dependencies.OSBuild = fmt.Sprintf("error: %s", err)
+	}
+	vd.ImageBuilder.Dependencies.OSBuild = strings.TrimSpace(out.String())
```

Comment:
Please use https://github.com/osbuild/images/blob/f52789ed6611ddf9b52ba0f06f7f70d5dd86c939/pkg/osbuild/osbuild-exec.go#L131-L146 instead of implementing it again here.